### PR TITLE
orca-c: init at git-2020-02-12

### PR DESCRIPTION
**Pull request by Sohalt**
##### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

File: pkgs/applications/audio/orca-c/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    patchShebangs tool
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install build/orca $out/bin/orca
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Orca is an esoteric programming language designed to quickly create procedural sequencers.";
```
Comment (ehmry) [inline pkgs/applications/audio/orca-c/default.nix:26]: Description must not start with the name of the package.

File: pkgs/applications/audio/orca-c/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    homepage = "https://github.com/hundredrabbits/Orca-c";
```
Comment (ehmry) [inline pkgs/applications/audio/orca-c/default.nix:27]: Nitpick: homepage is https://wiki.xxiivv.com/site/orca.html?

**Review by ghost — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/orca-c/default.nix (hunk 2 above)
Comment (Sohalt) [reply pkgs/applications/audio/orca-c/default.nix:27]: I think that's only the homepage for the "official" electron/JS implementation. The C implementation does not get linked from there.

**Comment by Sohalt** [comment]
Thanks!

**Pull request merged**